support/commercial.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: single
+title: Commercial Support
+meta_title: How to get WireMock commercial support with SLA
+description: Enterprise and commercial support and consulting options
+---
+
+WireMock is an open source project.
+In accordance with the [Apache License 2.0](https://github.com/wiremock/wiremock/blob/master/LICENSE.txt),
+in general there is no support or guarantees provided for it.
+You can get some assistance through WireMock community channels,
+and contribute to helping other users too.
+At the same time, there are vendors that provide commercial support for WireMock.
+
+## Products with commercial support
+
+### WireMock Cloud - Priority Support
+
+Built on WireMock, [WireMock Cloud](https://www.wiremock.io/) offers a hosted experience
+including a self-service UI, enterprise support, and unlimited scale.
+Key features, in addition to all WireMock capabilities,
+include support for manual and automated testing,
+importing APIs from OpenAPI and Swagger, and Chaos Engineering.
+
+For users of the [WireMock Cloud](https://www.wiremock.io/) service,
+WireMock Inc offers priority support with a guaranteed SLA as part the Enterprise subscription plan.
+You can find more info about WireMock Cloud support plans [here](https://www.wiremock.io/pricing).
+
+[Get in touch with our team](https://share-eu1.hsforms.com/1YSKnSP93Tsig1JoO3lXSawfbdiq) to discuss options or get a demo.
+
+### WireMocha - Integration for JetBrains IDEs
```

Comment:
Maybe everything from this point down belongs on a different page, since it's all 3rd party stuff?

Reply:
This is how we discussed it for the SUPPORT page, nothing changed since that

Reply:
Perhaps I misunderstood what was being proposed in that case, because I definitely think 3rd party stuff belongs on its own page(s).